src/chunk-stream.iced

```diff
@@ -2,53 +2,46 @@ stream = require('stream')
 
 exports.ChunkStream = class ChunkStream extends stream.Transform
 
-  # transform_func: accepts a buffer, returns a buffer. Used in the _trasform() method.
-  # block_size: the chunk size
-  # exact_chunking: boolean, specifies whether to pass chunks of exactly block_size or any multiple of block_size to transform_func
-  constructor : (@transform_func, @block_size, @exact_chunking, {@writableObjectMode, @readableObjectMode}) ->
+  constructor : (@transform_func, {@block_size, @exact_chunking, @writableObjectMode, @readableObjectMode}) ->
```

Comment:
I much prefer underscores, but the stream API expects variables in camel case. `writableObjectMode` and `readableObjectMode` get passed to the stream API. Should I change them all to camel?